gcloud/pubsub/topic.py

```diff
@@ -128,7 +123,7 @@ def create(self, client=None):
                        ``client`` stored on the current topic.
         """
         client = self._require_client(client)
-        client.connection.api_request(method='PUT', path=self.path)
+        client.connection.topic_create(topic_path=self.full_name)
```

Comment:
The slash-prefixed `path` is less useful, once we delegate building the actual payloads:  some of them need to embed the 'full_name' part into the resource in the body.